src/app.js

```diff
@@ -195,6 +195,10 @@ module.exports = App
 function run () {
   var self = this
 
+  modalDialogCustom.alert(`The Remix IDE has moved to https://remix.ethereum.org/.\n
```

Comment:
Should only display this if:
- `window.location` is not `file:`
- `window.location` is `http:` or `https:` and is not `remix.ethereum.org`